docs-chef-io/content/supermarket/install_supermarket.md

```diff
@@ -548,7 +548,7 @@ Encrypted S3 buckets are currently not supported.
 1. Shut down the server running Private Supermarket.
 1. Backup the `/var/opt/supermarket` directory.
 
-1. Download the [Chef Supermarket](https://downloads.chef.io/) package.
+1. Download the [Chef Supermarket](https://www.chef.io/downloads/tools/supermarket) package.
```

Comment:
This link is not working. Basically https://www.chef.io itself is not opening
<img width="1792" alt="Screenshot 2021-12-30 at 11 48 01 AM" src="https://user-images.githubusercontent.com/85236476/147726684-145fb7b3-5ad3-47f6-b7d3-8bbe43d848d5.png">

Reply:
@RajeshPaul38 It's working at my end.

Reply:
Loads fine for me. chef.io redirects to www.chef.io so this just avoids the extra time / effort spend handling the redirect